notebooks/233-blip-visual-language-processing/233-blip-visual-language-processing.ipynb

```diff
@@ -143,7 +143,7 @@
     }
```

Comment:
<p>there is no any transition between previous subarticle and interactive demo. Could ou please add (tatle, description, e.t.c.)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>